packages/core/src/browser/dialogs.ts

```diff
@@ -156,6 +169,7 @@ export abstract class AbstractDialog<T> extends BaseWidget {
 
         this.contentNode = document.createElement('div');
         this.contentNode.classList.add('dialogContent');
+        this.contentNode.setAttribute('style', `word-wrap: ${props.wordWrap}`);
```

Comment:
What if it is undefined? The same question for maxWidth?

Reply:
`        if (props.maxWidth === undefined) {`
            `container.setAttribute('style', 'max-width: none');`
        `}` 
`else {`
            `container.setAttribute('style', 'max-width: ${props.maxWidth} px');`
        `}
`
If maxWidth is undefined, it would result in the css property 'max-width: none' being applied to the dialog.

`        if (props.wordWrap !== undefined) {`
            `this.contentNode.setAttribute('style', 'word-wrap: ${props.wordWrap}');`
        `}`
If wordWrap is undefined, it would result in the css property 'word-wrap' not being applied to the dialog.